src/python_framework_v02/nhd_network.py

```diff
@@ -278,13 +283,13 @@ def reservoir_shore(connections, waterbody_nodes):
     for node in wbody_set:
         shore.update(filter(not_in, connections[node]))
     return list(shore)
-
+    LOG.debug("reservoir_shore - connections: %s", connections, "waterbody_nodes: %s", waterbody_nodes)
```

Comment:
This will likely raise an error. We're using the older string formatting language so it would look more like:
`LOG.debug("reservoir_shore - connections: %s waterbody_nodes: %s", connections, waterbody_nodes)`